src/main/java/com/splunk/logging/AckManager.java

```diff
@@ -39,26 +39,52 @@
 
   private static final ObjectMapper mapper = new ObjectMapper();
   private final HttpEventCollectorSender sender;
-  private final AckPollScheduler ackPollController = new AckPollScheduler();
+  private final PollScheduler ackPollController = new PollScheduler();
+  private final PollScheduler healthPollController = new PollScheduler();
   private final AckWindow ackWindow;
   private final ChannelMetrics channelMetrics;
 
   AckManager(HttpEventCollectorSender sender) {
     this.sender = sender;
     this.channelMetrics = new ChannelMetrics(sender);
     this.ackWindow = new AckWindow(this.channelMetrics);
+
+    // start polling for health
+    startPollingForHealth();
```

Comment:
Let's add a comment here about why we startPollingForHealth() and startPolling() at the same time (without waiting for a healthy healthcheck result first)